src/util/codec/error.rs

```diff
@@ -32,17 +31,28 @@ pub const ERR_DATA_OUT_OF_RANGE: i32 = 1690;
 quick_error! {
     #[derive(Debug)]
     pub enum Error {
-        Io(err: io::Error) {
+            Io(err: io::Error) {
```

Comment:
Generally error should be defined in the module root instead of util.

Reply:
This error is used in the module util.. where do your advised to define it then?

Reply:
Or maybe we could separate the error in module` util::codec` and  `coprocessor::codec`?

Reply:
Why not keep them as `coprocessor::Error`?